pages/apis/graphql/graphql_cookbook.md

```diff
@@ -678,6 +678,23 @@ mutation UpdateSessionDuration {
 }
 ```
 
+### Update the revoke inactive tokens after
+
+You can control how many days to revoke inactive API access tokens. By default that's `NEVER` revoke inactive API access tokens, but you can set to 30, 60, 90, 180, 365 days. This is only available to Enterprise customers.
```

Comment:
General tightening + I think it's good to be upfront about the Enterprise restriction:

```suggestion
On the Enterprise plan, you can control when inactive API access tokens are revoked. By default, they are never (`NEVER`) revoked, but you can set your token expiry policy to either 30, 60, 90, 180, or 365 days.
```